tests/BUILD

```diff
@@ -12,142 +12,137 @@ exports_files([
 
 load("//:subpar.bzl", "par_binary")
 
+# Configurations
+config_setting(
+    name = "subpar_test_python_version_2",
+    define_values = {
+        "subpar_test_python_version": "2",
+    },
+)
```

Comment:
I don't see this used anywhere, is that normal?  I'm a bit new to `config_setting`.

Reply:
Right the _2 is never used (the config_setting use the //conditions:default catch all condition), maybe consider making all select explicitely use that condition or simply delete that config_setting?